ParisBNF/abb/BNFabb87.xml

```diff
@@ -0,0 +1,213 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb87" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Chant collections</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="NV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno>BnF Éthiopien d'Abbadie 87</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title type="complete" ref="LIT4078Collectionch">Collection of antiphons</title>
+                            <textLang mainLang="gez"/>
+                            <note>Content description is based on 
+                                <bibl>
+                                    <ptr target="bm:Shelemay1993"/>
+                                    <citedRange unit="page">85</citedRange>
+                                    <citedRange unit="note">43</citedRange>
+                                </bibl>  
+                            </note>
+                            <msItem xml:id="ms_i1.1">
```

Comment:
if the parts occur regularly it would make sense to develop the record
LIT4078Collectionch
itroducing text divisions with labels 
e.g. 
```
<div type="textpart" xml:id="mazmur">
                        <label>mazmur antiphons</label>  
                     </div>
```
etc
and here refer to the derived IDs 
`<title type="complete" ref="LIT4078Collectionch#mazmur"/>
`etc

Reply:
@karljonaskarlsson plans to study it once in Paris, to be sure about the content.